### PR TITLE
Add more plugins.

### PR DESCRIPTION
**Pull request by Sven65**

### CODE REVIEW

File: Reddit/index.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+const Plugin = module.parent.require('../Structures/Plugin')
+const snekfetch = require("snekfetch")
+
+Object.defineProperty(Array.prototype, "random", {
```
Comment (Ratismal) [inline Reddit/index.js]: I don't like extending arrays because it breaks `for in` calls, which may break other plugins.

**Review by Ratismal — CHANGES_REQUESTED** [review]

**Comment by Snazzah** [comment]
Can you probably make these into [external commands](https://github.com/DiscordInjections/Plugins/tree/master/ExternalCommands)?

**Comment by Ratismal** [comment]
It's fine if he wants them to be standalone, I just don't want it to potentially break other plugins.

**Review by Ratismal — APPROVED** [review]

**Comment by Sven65** [comment]
Nice.

**Pull request merged**